packages/react/src/blocks/Image.tsx

```diff
@@ -331,7 +332,7 @@ class ImageComponent extends React.Component<any, { imageLoaded: boolean; load:
                   },
                 }),
               }}
-              loading="lazy"
+              loading={isPixel ? 'eager' : 'lazy'}
```

Comment:
TIL `lazy` is not used with JS disabled: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-loading

Reply:
oh ya JS is no the concern here, the concern is purely that we need the pixel to always eager load no matter what, even if not in the viewport

Reply:
that's interesting tho that lazy doesn't work with JS disabled, definitely unexpected

Reply:
yeah yeah, it's not related to the PR :D it's just a broken assumption of mine of how native lazy loading works